src/utils/apiPathObject.js

```diff
@@ -0,0 +1,27 @@
+import getSledilnikApiPath from '../helpers/getSledilnikApiPath';
+import getDateBefore from '../helpers/getDateBefore';
+
+const dateFrom = getDateBefore().toISOString();
+
+const pathsParamsObject = {
+  stats: { from: dateFrom },
+  patients: { from: dateFrom },
+  municipalities: { from: dateFrom },
+  hospitals: {},
```

Comment:
Your hospitals broke because the correct endpoint would be `hospitals_list`, remember to change it in `App.js` into `hospitals_listPath` as well, if this naming pattern remains.

Also, I think this file shouldn't exist either. Since `dateBefore` will end up being just a few lines, and `getSledilnikApiPath` will be preferably not exist, this file will end up being just an object, and that object can move to App.js, where it will be easier to grok and change.

Reply:
> Your hospitals broke because the correct endpoint would be hospitals_list, remember to change it in App.js into hospitals_listPath as well, if this naming pattern remains.

fixed

> Also, I think this file shouldn't exist either. Since dateBefore will end up being just a few lines, and getSledilnikApiPath will be preferably not exist, this file will end up being just an object, and that object can move to App.js, where it will be easier to grok and change.

in near future?

Reply:
Sure.